package swarm.msgs;

/// \ingroup swarm_msgs
/// \interface Neighbor_V
/// \brief A message containing a list of neighbor addresses.

message Neighbor_V
{
  /// \brief List of neighbors.
  repeated string neighbors = 1;

  /// \brief List of probabilities of messages getting through from each
  /// neighbor (parallel to neighbors list).
  repeated double neighbor_probabilities = 2;
}
